### PR TITLE
update owner for service bus

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -353,6 +353,8 @@
 
 # AzureSdkOwners: @kashifkhan
 # ServiceLabel: %Service Bus
+# ServiceOwners: @kashifkhan
```
Comment (kashifkhan) [inline .github/CODEOWNERS:356]: do we need `ServiceOwner`? Im definitely not on the service side :)

Comment (xiangyan99) [reply .github/CODEOWNERS:356]: Do we need "service attention" label?

Comment (kashifkhan) [reply .github/CODEOWNERS:356]: nope we dont.

**Review by kashifkhan — APPROVED** [review]
Thank you :)

**Pull request merged**